src/strategies/saddle-finance/index.ts

```diff
@@ -0,0 +1,136 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import remappedMerkleData from './remappedMerkleData.json';
```

Comment:
Could you load this file from IPFS directly? We would prefer not having large file within the library itself

Reply:
@bonustrack @ChaituVR done ✔️